uxarray/helpers.py

```diff
@@ -70,3 +95,14 @@ def determine_file_type(filepath):
             os._exit(0)
 
     return mesh_filetype
+
+
+def check_ugrid_cf_role(cf_role):
```

Comment:
Is this intended to be an internal or user API function? Also, please add it to corresponding docs index.

Reply:
This isn't used. Removing.